src/app/pages/HomePage/HomePage.tsx

```diff
@@ -15,9 +20,18 @@ export default function HomePage(): JSX.Element {
     refetch();
   }
 
+  function modalClick() {
+    setModal(!isModalOpen);
+  }
+
   return (
     <div className={style.pageContainer}>
-      <Header withLogo withBurgerButton className={style.header} />
+      <Header
+        withLogo
+        withBurgerButton
+        className={style.header}
+        onClick={() => modalClick()}
```

Comment:
```suggestion
        onClick={modalClick}
```

or remove lines 23 to 26 and do this
```suggestion
        onClick={() => setModal(!isModalOpen)}
```

Reply:
Resolved in [4c17b99](https://github.com/Seborider/plugs./pull/50/commits/4c17b99ff7e6a564e05c1e51fc680c1e93fc0ebc)